panther_analysis_tool/command/benchmark.py

```diff
@@ -35,8 +35,8 @@ def __str__(self) -> str:
 def run(  # pylint: disable=too-many-locals
     backend: BackendClient, args: argparse.Namespace
 ) -> Tuple[int, str]:
-    if not backend.supports_perf_test():
-        return 1, "benchmark is only supported via the api token"
+    if backend is None or not backend.supports_perf_test():
+        return 1, "Invalid backend: `benchmark` is only supported via API token"
```

Comment:
nit: `Invalid backend: benchmark` to me reads like "benchmark" is the name of the backend. I'd use `.` instead of `:`